### PR TITLE
Feature/preserve data (PR)

### PR DESCRIPTION
**Pull request by teke85**
# Project Requirements

We checked the documentation for Ruby's [JSON library](https://ruby-doc.org/stdlib-3.0.0/libdoc/json/rdoc/JSON.html).
On exit from our app we saved all the data (books, people, and rentals) into JSON files. One per type of data you are storing and named accordingly (e.g. rentals.json)
On startup of the app we loaded all data from files. We also remembered to handle the case when one or more of the files are missing.

### CODE REVIEW

File: lib/rental.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def initialize(date, book, person)
     @date = date
     @book = book
     @person = person
-    book.rentals << self
-    person.rentals << self
+    book['rentals'] << self
+    person['rentals'] << self
   end
 end
```
Comment (andersonlebon) [inline lib/rental.rb]: - [ ] Kindly make sure you fix this error when the rentals are being created. 

image

**Review by andersonlebon — COMMENTED** [review]
## Hi Team

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: lib/rental.rb (hunk 1 above)
Comment (EddyShimwa) [reply lib/rental.rb]: Thanks @andersonlebon  for reviewing this PR,

Comment (EddyShimwa) [reply lib/rental.rb]: let's do it

**Review by andersonlebon — APPROVED** [review]
## Hi @team @teke85 @EddyShimwa 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

## To highlight :champagne: 
- Clear syntax :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**